### PR TITLE
Skill - CatchingUp

### PR DESCRIPTION
**Pull request by JesterWizard**
If the foe doubles the skill holder, increase the skill holder's damage by the amount of speed the foe is about the doubling threshhold.

### CODE REVIEW

**Comment by MokhaLeee** [comment]
reviewed with comment

**Comment by JesterWizard** [comment]
Where's the comment?

**Comment by MokhaLeee** [comment]
Oh it's my fault forget to submit the review

File: Data/SkillSys/SkillInfo.c [hunk 1]
```diff
@@ -1292,4 +1292,11 @@ const struct SkillInfo gSkillInfos[0x100] = {
         .icon = GFX_SkillIcon_StanceSpectrum,
     },
 #endif
+
+#if (defined(SID_CatchingUp) && (SID_CatchingUp < MAX_SKILL_NUM))
+    [SID_CatchingUp] = {
+        .desc = MSG_SKILL_CatchingUp,
+        .icon = GFX_SkillIcon_WIP,
```
[the remark this answers is not in the document]
Comment (JesterWizard) [reply Data/SkillSys/SkillInfo.c:1299]: I don't have one to use. I'm not an artist. WIP is placeholder until that changes.

File: Wizardry/Core/BattleSys/Source/PreBattleCalc.c [hunk 2]
```diff
@@ -437,6 +437,19 @@ STATIC_DECLAR void PreBattle_CalcSkillsOnEnd(struct BattleUnit * attacker, struc
...
 
+#if (defined(SID_CatchingUp) && (SID_CatchingUp < MAX_SKILL_NUM))
+        if (SkillTester(unit, SID_CatchingUp))
+        {
+            /**
+             * Check if the enemy unit doubles the skill holder
+             * if there's any additional speed above the doubling threshold
+             * add that to the skillholder's attack
+             */
+            if((defender->battleSpeed - attacker->battleSpeed) >=4)
```
[the remark this answers is not in the document]
Comment (JesterWizard) [reply Wizardry/Core/BattleSys/Source/PreBattleCalc.c]: Done

**Pull request merged**